### PR TITLE
Libgrib_api buildscript

### PR DESCRIPTION
**Pull request by pspoerri**

### CODE REVIEW

File: package_builder/build_libgrib_api.sh [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    install_path_prefix_="${base_path}/install"
+else
+    install_path_prefix_="${install_dir}/libgrib_api"
+fi
+
+if [[ -z "${jasper_dir}" ]]; then
+    jasper_dir="${install_dir}/libjasper"
+fi
+
+cosmo_resources_path=${base_path}/cosmo_resources
```
Comment (jmbettems) [inline package_builder/build_libgrib_api.sh]: What is base_path? User defined?

File: package_builder/build_libgrib_api.sh [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            git clone $resources_repo "${cosmo_resources_path}"
+            if [ $? -ne 0 ]; then
+                exitError 4211 ${LINENO} "unable to obtain ${resources_repo}"
+            fi
+        fi
+        echo ">>> Checking out ${resources_version}"
+        pushd "${cosmo_resources_path}" &>/dev/null
+            git fetch
+            git checkout "${resources_version}"
+            if [ $? -ne 0 ]; then
```
Comment (jmbettems) [inline package_builder/build_libgrib_api.sh]: Git checkout origin/$version

**Comment by jmbettems** [comment]
export GRIB_DEFINITION_PATH=/users/bettems/projects/fieldextra/resources/grib_api_definitions_cosmo:/users/bettems/projects/fieldextra/resources/grib_api_definitions_vendor
export GRIB_SAMPLES_PATH=/users/bettems/projects/fieldextra/resources/grib_api_samples

File: package_builder/build_libgrib_api.sh [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+if [[ -z ${install_dir} ]]; then
+    exitError 4202 ${LINENO} "package install dir has to be specified"
+fi
+if [[ -z ${resources_version} ]]; then
+    exitError 4203 ${LINENO} "resources_version has to be specified (coupling to libgrib)"
+fi
+
+# Setup
+echo $@
+base_path=$PWD
```
Comment (pspoerri) [inline package_builder/build_libgrib_api.sh]: Actually, this base path is the current directory. Which can be different from the package dir.

File: package_builder/build_libgrib_api.sh (hunk 1 above)
Comment (pspoerri) [inline package_builder/build_libgrib_api.sh]: `base_path` is the current path we are located in.

File: package_builder/build_libgrib_api.sh [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+# Setup
+echo $@
+# The current directory
+base_path=$PWD
+setupDefaults
+
+if [[ ${install_local} == "yes" ]]; then
+    install_path_prefix_="${base_path}/install"
+else
+    install_path_prefix_="${install_dir}/libgrib_api"
```
Comment (lxavier) [inline package_builder/build_libgrib_api.sh]: add release number to the path

File: package_builder/build_libgrib_api.sh [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+fi
+
+if [[ -z "${jasper_dir}" ]]; then
+    jasper_dir="${install_dir}/libjasper"
+fi
+
+# Name of the COSMO Definitions Dir
+cosmo_definitions_dir=cosmo_definitions
```
Comment (lxavier) [inline package_builder/build_libgrib_api.sh]: bug

File: package_builder/build_libgrib_api.sh [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+    # Set fortran environment
+    export compiler=$1
+    setFortranEnvironment
+
+    # Set F77 compiler to F90
+    export F77=$FC
+
+    # Special override for Cray
+    if [ "${compiler}" == "cray" ]; then
+        export FCFLAGS="-g -hnoacc -e m -hfp_trap"
```
Comment (lxavier) [inline package_builder/build_libgrib_api.sh]: try if it work with 02

**Pull request merged**